IdokladSdk.IntegrationTests/Tests/Clients/Account/AccountTest.cs

```diff
@@ -197,5 +197,18 @@ public void UserUpdate_DoesNotFail()
             // Assert
             Assert.NotNull(data);
         }
+
+        [Test]
+        public void GetSubscriptionList_SubscriptionListGotSuccessfully()
+        {
+            // Act
+            var result = _accountClient.SubscriptionList()
+                .Sort(x => x.DateOfIssue.Desc())
+                .Get().AssertResult();
+
+            // Assert
+            Assert.NotNull(result);
+            Assert.AreEqual(4, result.TotalItems);
```

Comment:
Pokud si to dobře pamatuju, tak v tom předchozím PR jsi to testoval na 2 a předpokládám, že jsi pak kvůli tomu opravoval ty testy. Možná bych tady u těch operací vracejících seznamy testoval počet na >0. Teď to už nechej.